src/main/java/org/isf/lab/rest/LaboratoryController.java

```diff
@@ -347,15 +347,18 @@ public ResponseEntity<Boolean> deleteExam(@PathVariable Integer code) throws OHS
 	}
 
 	/**
-	 * Get all {@link LabWithRows}s.
+	 * Get the list of exams {@link LabWithRows}s divided by pages.
 	 * 
+	 * @param oneWeek
+	 * @param pageNo
+	 * @param pageSize
 	 * @return the {@link List} of found {@link LabWithRows} or NO_CONTENT otherwise.
 	 * @throws OHServiceException
 	 */
 	@GetMapping(value = "/laboratories", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<List<LabWithRowsDTO>> getLaboratory() throws OHServiceException {
+	public ResponseEntity<List<LabWithRowsDTO>> getLaboratory(@RequestParam boolean onWeek, @RequestParam int pageNo, @RequestParam int pageSize) throws OHServiceException {
```

Comment:
Why does the Javadoc have `oneWeek` and the method signature has `onWeek`?  Pick on or the other.